lib/MessageInterface.php

```diff
@@ -142,10 +146,15 @@ public function removeHeader(string $name): bool;
      *
      * Should be 1.0, 1.1 or 2.0.
      */
-    public function setHttpVersion(string $version);
+    public function setHttpVersion(string $version): void;
 
     /**
      * Returns the HTTP version.
      */
     public function getHttpVersion(): string;
+
+    /**
+     * Returns the HTTP version.
+     */
+    public function __toString(): string;
```

Comment:
I needed to do this because otherwise PHPstan complains:
```
------ ------------------------------------------------------------------------ 
  Line   lib/RequestDecorator.php                                                
 ------ ------------------------------------------------------------------------ 
  189    Call to an undefined method Sabre\HTTP\RequestInterface::__toString().  
 ------ ------------------------------------------------------------------------ 

 ------ ------------------------------------------------------------------------- 
  Line   lib/ResponseDecorator.php                                                
 ------ ------------------------------------------------------------------------- 
  76     Call to an undefined method Sabre\HTTP\ResponseInterface::__toString().  
 ------ ------------------------------------------------------------------------- 
```

The Decorator classes have `__toString` that expects __toString` to exist in a `RequestInterface` and `ResponseInterface`. And they both extend `MessageInterface`